jax/_src/lax/lax.py

```diff
@@ -1952,13 +1983,25 @@ def standard_abstract_eval(prim, shape_rule, dtype_rule, *args, **kwargs):
   least_specialized = _max(
       map(type, args), key=operator.attrgetter('array_abstraction_level'))
   if least_specialized is ConcreteArray:
-    return ConcreteArray(prim.impl(*[x.val for x in args], **kwargs))
+    out_vals = prim.impl(*[x.val for x in args], **kwargs)
+    if not prim.multiple_results:
+      out_vals = [out_vals]
+    out_avals = safe_map(ConcreteArray, out_vals)
   elif least_specialized is ShapedArray:
-    return ShapedArray(shape_rule(*args, **kwargs), dtype_rule(*args, **kwargs))
+    shapes, dtypes = shape_rule(*args, **kwargs), dtype_rule(*args, **kwargs)
+    if not prim.multiple_results:
+      shapes, dtypes = [shapes], [dtypes]
+    out_avals = safe_map(ShapedArray, shapes, dtypes)
   elif least_specialized is UnshapedArray:
-    return UnshapedArray(dtype_rule(*args, **kwargs))
+    dtypes = dtype_rule(*args, **kwargs)
+    if not prim.multiple_results:
+      dtypes = [dtypes]
+      out_avals = safe_map(UnshapedArray, dtypes)
```

Comment:
I think that the indentation is off here.

Reply:
Thanks for the catch!